ffi/framebuffer_SDL2_0.lua

```diff
@@ -15,6 +15,10 @@ function framebuffer:init()
     else
         self.bb = BB.new(600, 800)
     end
+    if os.getenv("EMULATE_BW_SCREEN") then
+        self.device.hasColorScreen = function() return false end
+        self.isColorEnabled = function() return false end
+    end
```

Comment:
I'm not quite sure if this should be here in base in this form?

Reply:
The idea here is that env specifically instructs fb to do all the emulation stuff directly, often at costs coercing frontends into things they have no idea about. So its either this or polluting frontend with debug brutalism (test spec are fine with this too, as opposed to *reading* funky flags we'd prefer to not shim).

Reply:
Alright, it's a niche development option anyway. :-)